popup.html

```
@@ -10,35 +10,42 @@
-      <h1 class="title">TabbyMcTabface</h1>
-      <p class="subtitle">Tab Management with Attitude</p>
+      <img class="mascot" src="icons/icon128.png" width="72" height="72" alt="Skeptical Wombat mascot">
+      <div class="brand-copy">
+        <h1 class="title"><span>Tabby</span><span class="brand-mc">Mc</span><span class="brand-tabface">Tabface</span></h1>
+        <p class="subtitle">Tab management under skeptical supervision</p>
+      </div>
     </header>
 
     <!-- Main Actions -->
     <section class="actions">
       <button id="feelingLuckyBtn" class="btn btn-primary">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)